drv/gimlet-seq-server/src/main.rs

```diff
@@ -513,6 +518,19 @@ cfg_if::cfg_if! {
             sys.gpio_reset(UART_TX_ENABLE).unwrap();
         }
 
+        fn uart_sp_to_sp3_disable() {
+            let sys = sys_api::Sys::from(SYS.get_task_id());
+
+            sys.gpio_configure_output(
+                UART_TX_ENABLE,
+                sys_api::OutputType::PushPull,
+                sys_api::Speed::High,
```

Comment:
```suggestion
                sys_api::Speed::Low,
```
😉

Reply:
hehe, I can make that change, probably also in the enable function which I copied to make this?

Reply:
Sure, that makes sense!